### ISSUE TITLE
[Card group, Card in Card]: Cannot click card to navigate to new page - React wrapper Safari mobile

### ISSUE THREAD

**Issue by RichKummer** [carbon-design-system/carbon-for-ibm-dotcom#8581]
### Description

On Safari mobile with VoiceOver using the React wrapper build, I cannot double click on a card to navigate to the linked page (Example domain page). Tapping multiple times will trigger a page preview in Safari. Compared to the current Web Component version, which does navigate with VoiceOver on.

Card group
![IMG_2073](https://user-images.githubusercontent.com/15643582/159297692-82ecbe5a-33c8-4d97-83a8-48f4048287c2.PNG)

Card in card
![IMG_2072](https://user-images.githubusercontent.com/15643582/159297680-101c8362-1188-4c8b-83b1-3b91f7669774.PNG)

### Component(s) impacted

Card group
Card in card
Card link

### Browser

Safari

### Carbon for IBM.com version

v1.17.0-rc.1

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

C4IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://codesandbox.io/s/github/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/web-components/examples/codesandbox/components-react/card-group

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by kennylam** [comment]
@RichKummer I just spoke to @ariellalgilmore about this and we actually don't think it's a bug at all. In the mobile context the user needs to be able to swipe through each element, which will announce its role in the context of the landmark. In this case, there is in fact only one link, which is the text CTA. Clicking the eyebrow/header wouldn't make sense, as they are not actually links and were never announced that way. Thoughts?

@oliviaflory Happy to keep discussion going, but like I said this is behaving correctly as far as screen readers go. I lowered the severity and moved to `pending scheduling`.

**Comment by oliviaflory** [comment]
@kennylam makes sense to me! Thanks for looking into it, should we close because it's behaving as expected?

**Comment by kennylam** [comment]
@oliviaflory Yep, sounds good!
